contrib/traffic-management-plugins/README.md

```diff
@@ -0,0 +1,155 @@
+Wasabi installation.
+
+The Traffic Estimator plugin helps with the job of correctly setting the sampling percentages for
+experiments that are related to each other through mutual exclusion and that will,
+generally, be used through the "batch assignment" API
+(e.g., /api/v1/assignments/applications/CTG/pages/page1/users/userID1) .  When you use
+mutual exclusion, experiments that are of a higher priority (see the Priority tab)
+will take traffic away from experiments that they are mutually exclusive to that are
+of a lower priority.  In order to compensate for this, if you want one of those lower
+priority experiments to have a specific "target sampling percentage" of the total traffic,
+you need to increase the "experiment sampling percentage" (the sampling percentage configured
+in the UI for that lower priority experiment).  The Traffic Estimator helps you
+make such compensative calculations and even to set the sampling percentages with one button click.
+
+Similarly, the Traffic Analyzer plugin helps you monitor the actual percentages of
+user traffic assigned to each experiment related by mutual exclusion.  In other words,
+once you have used the Traffic Estimator to set the correct sampling percentages into your
+experiments, and have let them run for a day or two, you can then use the Traffic Analyzer
+to automatically retrieve daily assignment results for the related experiments and
+confirm that you are getting the desired target sampling percentages.
+
+##Installation
+
+Simply copy the plugins directory (which contains 3 other directories) from this directory
+into the modules/ui/app directory.  
+
+Next, look at the file modules/ui/app/scripts/plugins.js .  If it only has the following
+in it:
+
+```
+var wasabiUIPlugins = [];
+```
+
+Then you can simply replace it with the file scripts/plugins.js in this directory.
+
+NOTE: If the file already contains some other plugin definitions, you will need to
+take the definition objects from the scripts/plugins.js file in this directory and
+add them to the list in the wasabiUIPlugins array in the modules/ui/app/scripts/plugins.js file.
+
+Now, simply restart your Wasabi server, which also serves the UI.  
+If you are using the wasabi.sh script, that
+would consist of:
+
+```
+./bin/wasabi.sh stop:wasabi start:wasabi
+```
+
+(NOTE: If you are running the Wasabi UI locally, e.g., using "grunt serve", simply stop
+that command and run it again.)
+
+To confirm successful installation of the plugins, login to the UI and click on the
+Plugins tab.  You should see descriptions of the two plugins and buttons to launch them.
+
+##Usage
+
+When you have successfully installed the plugins and restarted the Wasabi server, you
+can login and click on the Plugins tab to use the plugins.  You will see descriptions
+of the two plugins and a button labeled Run next to each.  To use a plugin, simply
+click on the Run button and a dialog will appear with the UI for the plugin.
```

Comment:
This is the first step in the Installation section of the README for the plugins:

Simply copy the plugins directory (which contains 3 other directories) from this directory into the modules/ui/app directory.

Did you do that?  If you only moved the plugins.js file, then it is telling the UI that you have these plugins, and hence, the buttons for them get created and shown, but the code to actually implement the UI for them hasn't been put in a place where they can be loaded and run, e.g., in the modules/ui/app directory.  I guess we could have added more error handling in the plugin system?  But I hope that if you follow the instructions in the README, it will work better for you?